control-plane-operator/main.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane"
 	"github.com/openshift/hypershift/support/releaseinfo"
 
+	"sigs.k8s.io/cluster-api/controllers/remote"
```

Comment:
Please let's don't introduce an unnecessary dependency here and have our own helper func for this.